Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM node:current-alpine
+USER root
+
+RUN apk add --no-cache --update git \
```

Comment:
why we need to git clone cctail and cant run just "npm install cctail"

Reply:
Cause we need cctail as an application(program) node and for that application we are doing additional npm install inside cctail(they have their own package.json).
We are running our program in supervisord.conf line 23

```
[program:cctail]
command=sh -c "node cctail/dist/cctail.js %(ENV_CCTAIL_ARGS)s | awk '{ print strftime(\"cctail:\ %%Y-%%m-%%d\ %%H:%%M:%%S\ -\"), $0 }'";
redirect_stderr=true
stdout_logfile=/dev/stdout
stdout_logfile_maxbytes=0
```

@Doron-Bargo